src/magento2/harness.yml

```diff
@@ -130,6 +133,9 @@ attributes:
         - run "magento config:set 'web/secure/use_in_frontend'  '1'"
         - run "magento config:set 'web/secure/use_in_adminhtml' '1'"
         - run "magento config:set 'web/seo/use_rewrites'        '1'"
+        - if [[ "$HAS_VARNISH" == "true" ]]; then
+            run "magento config:set 'system/full_page_cache/caching_application' '2'";
```

Comment:
isn't there a few other config to set as well in this case? (like host port maybe?)

Reply:
Nope, I think they are just for the VCL generator

Reply:
ah I see :+1: